src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10% auto;
+  width: getCalendarWidth($day-box-size, $calendar-paddings);
```

Comment:
Ensure that the function `getCalendarWidth` is defined elsewhere in your SCSS files. This function is used to calculate the width of the calendar, and if it's not defined, it will cause an error during compilation.